FreeRTOS/Test/CMock/tasks/tasks_1_utest.c

```diff
@@ -3550,6 +3550,29 @@ void test_vTask_Set_Get_ThreadLocalStoragePointer_success( void )
     TEST_ASSERT_EQUAL_PTR( pValue, ret_pValue );
 }
 
+/* this test ensures that if the index is negative, then NULL is returned. */
```

Comment:
I think it makes sense to just skip this test case when building with sanitizers enabled.